bin/parse_pom_xml

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby
+
+# Write a JSON string of the dependencies of a Java Maven project
+# to stdout using its pom.xml project file
+
+require 'json'
+
+dependencies_regex = %r{
+    <dependencies>
+        [\s\S]*
+    <\/dependencies>
+}x
+
+dependency_metadata_regex = %r{
```

Comment:
This regex assumes the keys follow a certain order, but the actual order could be random.
In addition, it assumes the file follows a specific format (with the newlines, spaces, etc.) but files that do not follow this format could be valid too.

Reply:
There is likely existing code that parses xmls.  Could you google it?

Reply:
Is it possible for a pom to not have some of the keys in the regex below and still be valid?

Reply:
It's my understanding that the only required keys are `groupId` and `artifactId`.